rails-time-track/db/seeds.rb

```diff
@@ -12,7 +12,7 @@
   
   manager1 = User.create(
     name: "Brayan Manager",
-    email: "linzeur@hotmail.com",
+    email: "linzeur@hotmail.com",j
```

Comment:
Something that should not be with your clean code.

Reply:
Oh, thanks a lot man!! Right now, I am going to delete that